iac/terraform/main.tf

```
@@ -0,0 +1,9 @@
+
+module "storage" {
+  source = ".//storage"
+}
+
+module "container" {
+  source = ".//container"
```

```suggestion
  source = "./container"
```
is there a reason for .//